conf/modules.config

```diff
@@ -57,11 +57,32 @@ process {
         ]
     }
 
+    withName: ENSEMBLVEP_VEP {
+        container = 'ensemblorg/ensembl-vep:release_111.0' //sticking to v111 for now, but we should update this
+        def args_list =  [
+            "--offline",
+            "--format vcf",
+            "--vcf",
+            "--xref_refseq",
+            "--variant_class",
+            "--numbers",
+            "--hgvs",
+            "--hgvsg",
+            "--canonical",
+            "--symbol",
+            "--flag_pick",
+            "--no_stats",
+            "--fields \"Allele,Consequence,IMPACT,SYMBOL,Feature_type,Gene,PICK,Feature,EXON,BIOTYPE,INTRON,HGVSc,HGVSp,STRAND,CDS_position,cDNA_position,Protein_position,Amino_acids,Codons,VARIANT_CLASS,HGVSg,CANONICAL,RefSeq\""
+        ]
+        ext.args = args_list.join(" ")
+        ext.prefix =  {"variants.${meta.id}.vep"}
+    }
```

Comment:
These  arguments will be added to the vep command. To ease the comparison, I copy-pasted final vep commands in vcf headers obtained running the pipeline on the main branch and this branch. This way it is easier to review the final command.

On the main branch, we see this command (reformatted to put one option per line):
```
vep 
  --cache 
  --canonical 
  --compress_output bgzip 
  --database 0 
  --dir vep 
  --dir_plugins /plugins 
  --fasta [PATH]/Homo_sapiens_assembly38.fasta 
  --fields Allele,Consequence,IMPACT,SYMBOL,Feature_type,Gene,PICK,Feature,EXON,BIOTYPE,INTRON,HGVSc,HGVSp,STRAND,CDS_position,cDNA_position,Protein_position,Amino_acids,Codons,VARIANT_CLASS,HGVSg,CANONICAL,RefSeq 
  --flag_pick 
  --fork 4 
  --format vcf 
  --hgvs 
  --hgvsg 
  --input_file family2.splitted.vcf.gz 
  --no_stats 
  --numbers 
  --offline 
  --output_file variants.family2.vep.vcf.gz 
  --symbol 
  --variant_class 
  --vcf 
  --xref_refseq'
```

Reply:
I think those are some good changes I just don't understand why the fork is being set to 2 when it seems to be 4 in the nextflow.config file, under the ENSEMBL_VEP process definition?

Reply:
Bonne question. Je crois que c'est ce paramètre dans le test config qui crée ça:  `max_cpus   = 2`

Voici ce qui est configuré dans nextflow.config pour le process ENSEMBL_VEP:
cpus	=	{ check_max( 4 		* task.attempt, 'cpus'		)	}

Dans la fonction check_max, la valeur 4 va être remplacée par  `Math.min( obj, params.max_cpus as int )`